internal/sqladapter/exql/column.go

```diff
@@ -51,7 +51,9 @@ func (c *Column) Compile(layout *Template) (compiled string) {
 
 		for i := range nameChunks {
 			nameChunks[i] = trimString(nameChunks[i])
-			nameChunks[i] = mustParse(layout.IdentifierQuote, Raw{Value: nameChunks[i]})
+			if nameChunks[i] != "*" {
```

Comment:
I like this, but could you invert the condition?:

```go
if nameChunks[i] == "*" {
  continue
}
nameChunks[i] = mustParse(layout.IdentifierQuote, Raw{Value: nameChunks[i]})
```

Code is equivalent but I think using `==` is better here because it `*` is a special case.